gameanalytics/SCsub

```diff
@@ -6,7 +6,7 @@ sources = [
 ]
 
 module_env = env.Clone()
-module_env.Append(CXXFLAGS=['-std=c++11'])
+module_env.Append(CXXFLAGS=['-std=c++17'])
```

Comment:
I suggest removing it instead if the module doesn't require C++17 specifically.

BTW `godot-3.3-stable` uses `-std=gnu++14` (C++14).

Reply:
I have remove the line now